packages/openapi-v3/src/decorators/response.decorator.ts

```diff
@@ -0,0 +1,83 @@
+// Copyright IBM Corp. 2018,2019. All Rights Reserved.
+// Node module: @loopback/openapi-v3
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+import {MethodMultiDecoratorFactory} from '@loopback/core';
+import * as httpStatus from 'http-status';
```

Comment:
It should be `import httpStatus from 'http-status'` now.

Reply:
I'll make these changes today